WordPress/src/main/res/values/ids.xml

```diff
@@ -1,19 +1,18 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <item type="id" name="row_post_id" />
```

Comment:
I think we should consider reverting this change is it might cause unnecessary conflicts. Moreover, the `wordpress-mobile:feature/post-filters-compat-view-toggle ` working branch won't make any other changes but formatting to the ids.xml after we merge this PR.